HW8/Recursion.cpp

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+
+int factorial_Recursion(int result, int count)
+{
+	if (count == 1)
+	{
+		return result;
+	}
+
+	result = result * --count;
+
+	factorial_Recursion(result, count);
+}
+
+void printNaturalNumbers_To_Smallest_Recursion(int count)
+{
+	if (count == 0)
+	{
+		return;
+	}
+
+	std::cout << count << std::endl;
+	printNaturalNumbers_To_Smallest_Recursion(--count);
+}
+
+void printNaturalNumbers_To_Largest_Recursion(int count, int current = 1)
```

Comment:
can be implemented without additional variable `current`

Reply:
Corrected